buildSrc/src/main/java/com/uber/okbuck/core/annotation/AnnotationProcessorCache.java

```diff
@@ -114,8 +115,8 @@ public AnnotationProcessorCache(
    */
   @SuppressWarnings("BooleanMethodIsAlwaysInverted")
   public boolean hasEmptyAnnotationProcessors(Project project, String configurationString) {
-    Configuration configuration = getConfiguration(project, configurationString);
-    return hasEmptyAnnotationProcessors(project, configuration);
+    Optional<Configuration> configuration = getConfiguration(project, configurationString);
+    return configuration.map(files -> hasEmptyAnnotationProcessors(project, files)).orElse(true);
```

Comment:
```suggestion
    return configuration.map(config -> hasEmptyAnnotationProcessors(project, config)).orElse(false);
```
should be false if configuration is not present